src/Stormpath/DataStore/DefaultDataStore.php

```diff
@@ -292,12 +292,14 @@ private function toStdClass(Resource $resource, $customData = false)
 
             $property = $resource->getProperty($name);
 
+            $nameIsCustomData = $name == Resource::CUSTOMDATA_PROP_NAME;
```

Comment:
Please get the constant from the `CustomData` class, after adding it there, of course.